protobuf/transform.go

```diff
@@ -130,13 +130,13 @@ func (t *Transformer) transformTypeList(pkg *Package, types []scanner.Type, name
 		return NewGeneratedNamed(toProtobufPkg(pkg.Path), msgName)
 	}
 
-	return t.transformType(pkg, types[0])
+	return t.transformType(pkg, types[0], &Message{}, &Field{})
```

Comment:
We could add a `nil` check in decorator{,s}.Run, I'm not a fan of adding the `nil` checks in the decorators themselves.